.gitignore

```diff
@@ -341,4 +341,8 @@ src/test/python/output/
 src/test/python/__pycache__/
 
 # Ignore copied license
-src/package/vsix/LICENSE
\ No newline at end of file
+src/package/vsix/LICENSE
+
+build
+*.winmd
+*_h.h
```

Comment:
Are we adding this ignore because you're generating output into the source directories? This really isn't a good idea

Reply:
The old test projects rely on these files being created in the respective project dirs. This is the typical way that these msbuild-based projects work. I didn't want to mess with them until I have more time to clean then up.

Reply:
you could either pass the cmake_binary_dir in to your project and use that to configure build output outside the src tree, or take a deferral and clean this up later (open a work item)

Reply:
Done: https://github.com/Microsoft/xlang/issues/195

Reply:
Actually I'll get this fixed right away.

Reply:
fixed